app/forms/openid_connect_authorize_form.rb

```diff
@@ -129,10 +129,6 @@ def requested_aal_value
       Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF
   end
 
-  def biometric_comparison_requested?
-    !!parsed_vectors_of_trust&.any?(&:biometric_comparison?)
-  end
-
   def parsed_vectors_of_trust
```

Comment:
nit: consider making this private

Reply:
good idea! updated in https://github.com/18F/identity-idp/pull/11007/commits/47f1f394ddc3889599171344f34d7eb46af97985